src/test/regress/expected/create_index.out

```diff
@@ -3129,3 +3129,70 @@ explain (costs off)
  Optimizer: Postgres query optimizer
 (4 rows)
 
+--
+-- Check spgist index work with AO tables
+--
+-- insert to AO table with spgist index
+create table spgist_ao_test(idx_col int4range)
```

Comment:
In this file all other SQL words are in capital (**CREATE** but not **create**).
Should not it be consistent with the rest of the file?